src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1018,16 +995,15 @@
   (let [csr-path (path-to-cert-request csrdir subject)]
     (if (fs/exists? csr-path)
       (if (fs/delete csr-path)
-        (let [msg (str "Deleted certificate request for " subject)]
```

Comment:
The message is being passed to log, and that one should be `trs` for sure.  The next line after that, though, is returning a map literal as the result of this fn, and that map literal contains the message.  So if that is bubbling up into the HTTP response via liberator then we're in another situation where we need both `trs` and `tru`.

In any case I'm OK with punting that audit to a follow-up ticket.